install/management/appliance.go

```diff
@@ -301,18 +279,22 @@ func (d *Dispatcher) createAppliance(conf *configuration.Configuration) error {
 			Value: fmt.Sprintf("-serveraddr=0.0.0.0 -port=%s -port-layer-port=8080 %s", d.DockerPort, d.dockertlsargs)})
```

Comment:
Shouldn't all of these also be in the vch config, so that they can be serialized with the extraconfig package?